chef/data_bags/crowbar/migrate/provisioner/201_chef_splay.rb

```diff
@@ -0,0 +1,9 @@
+def upgrade(ta, td, a, d)
+  a["chef_splay"] = ta["chef_splay"]
```

Comment:
if we dont have a guard here and we backport this migration, it would erase the current configured value with the default for an environment which was updated from soc7 to soc8